### ISSUE TITLE
Plugin won't start, Result::unwrap() panick

### ISSUE THREAD

**Issue by hardtopnet** [theCapypara/steamdeck-decky-syncthing#16]
I tried using the latest release on the Decky store (0.2.0), I get the infinite loading animation.

The watchdog log seems to be reporting a syncthing config deserialization issue

```
SerdeJson(Error("invalid type: string \"8384\", expected u32", line: 1, column: 211))
``` 

8384 seemingly being the port to the HTML Web UI.

Full logs: [decky-syncthing.zip](https://github.com/user-attachments/files/17740505/decky-syncthing.zip)

### Versions
| Soft| Version|
|--------|--------|
|OS|SteamOS v3.6.20|
|Decky Loader|v3.0.4| 
|Syncthing GTK|me.kozec.syncthingtk v0.9.4.5|

Just in case, has there been an unreleased fix on this repo for this issue ?

**Comment by hardtopnet** [comment]
excerpt from my syncthing config file (\home\deck\.var\app\me.kozec.syncthingtk\config\syncthing\config.xml)

![image](https://github.com/user-attachments/assets/cc91abd3-0d6e-41be-9fb3-a0086f0c45d5)

**Comment by theCapypara** [comment]
Hi! This is really odd. This is actually about the configuration file of the plugin itself.
You can find this file at `homebrew/settings/decky-syncthing/decky-syncthing.json`. Change `port` to have the value `8384` instead of `"8384"` to resolve this.

Now weirdly, this shouldn't be an issue, the UI should save this as an integer. I used to have a fallback in the backend, I might add this back in...

**Comment by hardtopnet** [comment]
**Yep, that did the trick...**

I just noticed something else, but it's unrelated. Tell me if it's worth opening a new issue.

I had already installed **SyncThing GTK** (`me.kozec.syncthingtk`) as a Flatpak, and upon **Decky SyncThing** startup I specified `me.kozec.flatpak is already installed`.
- I got an error stating said package wasn't installed, and **Decky SyncThing** couldn't start it (when it's installed and already running)
- I selected the first option `SyncThing is installed as a service from pacman` which worked.

> note: I had registered syncthing as a service that starts on boot before installing **Decky SyncThing**.
